lib/middleware/request_tracker.rb

```diff
@@ -118,6 +139,8 @@ def self.get_data(env, result, timing, request = nil)
     is_message_bus = request.path.start_with?("#{Discourse.base_path}/message-bus/")
     is_topic_timings = request.path.start_with?("#{Discourse.base_path}/topics/timings")
 
+    has_deferred_track_header = %w[1 true].include?(env["HTTP_DISCOURSE_DEFERRED_TRACK_VIEW"])
```

Comment:
Maybe just a small note here about how this header is set via message bus? I see you added it above in the `log_request` method, but in here we added a note about `discourse-track-view`, so it would be good to do the same for `discourse-deferred-track-view`.

Reply:
Added :+1: